### DISCUSSION TITLE
what is replacement of failOnStatusCode in cypress 10

### DISCUSSION THREAD

**Discussion by techshashank80** [cypress-io/cypress#23759] [Questions and Help]
Hi  there,

I was using failOnStatusCode=false but after upgrading to cypress it started giving errors. could you please let me know what is replacement for his in cypress 10?

Thanks in advance!!

**Comment by emilyrohrbough** [comment]
This shouldn't have changed. If you are having issues, please open an issue with details.

**Reply by techshashank80** [reply]
it was earlier in cypress.json with 9.x.x version now its in cypress. config.ts. Its giving error that it does not exists.
<img width="859" alt="image" src="https://user-images.githubusercontent.com/34492336/189977659-f7637a8e-47cc-43d6-be39-32b6f0e72b6d.png">

**Comment by mikespinks0401** [comment]
@techshashank80 any updates?
